.github/workflows/production-suite.yml

```diff
@@ -77,3 +77,90 @@ jobs:
         with:
           name: production-tarball
           path: /tmp/production-build
+
+  production_install:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - docker_image: mepriyank/actions:bionic
+            name: Bionic Production Install
+            is_bionic: true
+            os: bionic
+
+          - docker_image: mepriyank/actions:focal
+            name: Focal Production Install
+            is_focal: true
+            os: focal
+
+    name: ${{ matrix.name  }}
+    container: ${{ matrix.docker_image }}
+    runs-on: ubuntu-latest
+    needs: production_build
+    env:
+      # GitHub Actions overrides HOME to /github/home/ but we want
+      # undo the override so it points to /home/github.
+      HOME: /home/zulip
+
+    steps:
+      - name: Download built production tarball
+        uses: actions/download-artifact@v2
+        with:
+          name: production-tarball
+          path: /tmp
+
+      - name: Add required permissions and setup
+        run: |
+          # This is the GitHub Actions specific cache directory the
+          # the current github user must be able to access for the
+          # cache action to work. It is owned by root currently.
+          sudo chmod -R 0777 /__w/_temp/
+
+          # Create the zulip directory that the tools/ci/ scripts needs
+          mkdir -p /home/github/zulip
+
+          # Since actions/download-artifact@v2 loses all the permissions
+          # of the tarball uploaded by the upload artifact fix thoses.
+          chmod +x /tmp/production-extract-tarball
+          chmod +x /tmp/production-upgrade-pg
+          chmod +x /tmp/production-install
+          chmod +x /tmp/production-verify
+
+      - name: Create cache directories
+        run: |
+          dirs=(/srv/zulip-{npm,venv,emoji}-cache)
+          sudo mkdir -p "${dirs[@]}"
+          sudo chown -R github "${dirs[@]}"
+
+      - name: Restore node_modules cache
+        uses: actions/cache@v2
+        with:
+          path: /srv/zulip-npm-cache
+          key: v1-yarn-deps-${{ matrix.os }}-${{ hashFiles('package.json') }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: v1-yarn-deps-${{ matrix.os }}
+
+      - name: Do Bionic hack
+        if: ${{ matrix.is_bionic }}
+        run: |
+          # Temporary hack till `sudo service redis-server start` gets fixes in Bionic. See
+          # https://chat.zulip.org/#narrow/stream/3-backend/topic/Ubuntu.20bionic.20CircleCI
+          sudo sed -i '/^bind/s/bind.*/bind 0.0.0.0/' /etc/redis/redis.conf
```

Comment:
Commit message talks about "the memcached hack step". Should that instead be "the redis hack step"?

Reply:
No, what is mentions is correct though commit message needs some tweaks. It talks about removing the `Memcached hack` step that was present in CircleCI that Anders pointed [out earlier](https://github.com/zulip/zulip/pull/15843#discussion_r458340587). I will make the commit message more clear.

Reply:
I updated the commit messaged to make this clear.